<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CircularWidget</class>
 <widget class="QWidget" name="CircularWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>282</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QPushButton" name="markInitPositionButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Marcar Posição Inicial</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" rowspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="xinitSpinBoxx">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <double>850.000000000000000</double>
       </property>
       <property name="value">
        <double>200.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="yinitSpinBoxx">
       <property name="maximum">
        <double>750.000000000000000</double>
       </property>
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="zinitSpinBoxx"/>
     </item>
    </layout>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="markFinalPositionButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>Marcar Posição Central</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <layout class="QGridLayout" name="gridLayout_28">
     <item row="0" column="0">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>X:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="xfinalSpinBoxx">
       <property name="maximum">
        <double>850.000000000000000</double>
       </property>
       <property name="value">
        <double>500.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>Y:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="yfinalSpinBoxx">
       <property name="maximum">
        <double>750.000000000000000</double>
       </property>
       <property name="value">
        <double>60.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>Z:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="zfinalSpinBoxx"/>
     </item>
    </layout>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_29">
     <property name="text">
      <string>Unidade:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="initFinalPosUnitComboBox">
     <property name="sizeIncrement">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <item>
      <property name="text">
       <string>mm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>cm</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>m</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>pulsos</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>&quot;</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_35">
     <property name="text">
      <string>Ângulo ao redor do eixo X:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <layout class="QGridLayout" name="gridLayout_26">
     <item row="0" column="0">
      <widget class="QDoubleSpinBox" name="xangleSpinBoxx">
       <property name="minimum">
        <double>-90.000000000000000</double>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string>º</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_27">
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="vAngleSpinBoxx">
       <property name="minimum">
        <double>-90.000000000000000</double>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>º</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_49">
       <property name="text">
        <string>Orientação vertical da tocha:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_29">
     <item row="0" column="2">
      <widget class="QLabel" name="label_52">
       <property name="text">
        <string>º</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="hAngleSpinBoxx">
       <property name="minimum">
        <double>-90.000000000000000</double>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_51">
       <property name="text">
        <string>Orientação horizontal da tocha:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>markInitPositionButton</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget_2</zorder>
  <zorder>layoutWidget_3</zorder>
  <zorder>layoutWidget_4</zorder>
  <zorder>label_35</zorder>
  <zorder>layoutWidget_5</zorder>
  <zorder>label_29</zorder>
  <zorder>markFinalPositionButton</zorder>
  <zorder>initFinalPosUnitComboBox</zorder>
  <zorder>label_50</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
